src/Nexucis/Elasticsearch/Helper/Nodowntime/IndexHelper.php

```diff
@@ -700,20 +702,14 @@ protected function existsAlias($alias)
             'name' => urlencode($alias)
         );
 
-        $response = $this->client->indices()->existsAlias($params);
-
-        if (is_array($response) && array_key_exists('status', $response)) {
```

Comment:
a bit weird you removed this code and it is still working. Was it a dead code ? Or am I missing something new in ElasticSearch ?

Reply:
`existsAlias` always return bool, so it was a dead code